seed/mappings/mapper.py

```diff
@@ -1,7 +1,6 @@
 """
 :copyright: (c) 2014 Building Energy Inc
```

Comment:
It's correct in the upstream branch 

https://github.com/SEED-platform/seed/blob/develop/seed/mappings/mapper.py

This branch was just made prior to that change.